src/praisonai/praisonai/framework_adapters/autogen_adapter.py

```
@@ -160,10 +270,103 @@ def run(self, config: Dict[str, Any], llm_config: List[Dict], topic: str) -> str
+            results = []
+            for task in combined_tasks:
+                chat_result = user_proxy.initiate_chat(
+                    manager,
+                    message=task
+                )
+                if hasattr(chat_result, 'summary'):
+                    results.append(chat_result.summary)
+                else:
+                    results.append(str(chat_result))
+            
+            return f"### AG2 Output ###\n" + "\n\n".join(results)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)